api/v1alpha1/release_types.go

```diff
@@ -138,128 +114,128 @@ type Release struct {
 	Status ReleaseStatus `json:"status,omitempty"`
 }
 
-// HasStarted checks whether the Release has a valid start time set in its status.
```

Comment:
alphabetizing this list of function makes the review process so much harder because github can't show the diff properly at all

Reply:
I agree. Next time I'll have a single commit for the reordering.